docs/source-pytorch/common/lightning_module.rst

```diff
@@ -1065,22 +1065,19 @@ for more information.
 
 .. code-block:: python
 
+    # devices can be GPUs, TPUs, ...
+    @on_every_device
```

Comment:
that's a little confusing 😄

Reply:
What do you suggest instead? Or do you want to remove it?

Reply:
maybe a comment rather than a decorator?